deepgram/_types.py

```diff
@@ -63,13 +63,20 @@ class TranscriptionOptions(TypedDict, total=False):
     version: str
     multichannel: bool
     alternatives: int
+    # numerals will be deprecated in the future
     numerals: bool
+    numbers: bool
+    numbers_spaces: bool
     search: List[str]
     callback: str
     keywords: List[str]
     ner: str
     tier: str
-    replace: str
+    dates: str
+    date_format: str
+    times: str
+    dictation: str
+    measurements: str
```

Comment:
Are we removing the `replace` parameter? 

Also, I wanted to double-check check but should `dates, times, dictation, measurements` all be booleans and not strings? 

Lastly, are we missing `smart_format` per the documentation?